{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/js/frontend-editing/wysiwyg.js

```diff
@@ -0,0 +1,95 @@
+import tinymce from 'tinymce/tinymce'
```

Comment:
I think there is probably a better answer than having a second copy of TinyMCE running on the frontend - we should be able to change TinyMCE settings in a single place (`settings/base.py`) rather than two.

Reply:
(The other option is that we have an extremely pared-down version of MCE for small edits - in which case, you can remove most of the plugins that have been loaded.)